components/Socials/sub-components/SocialType.jsx

```diff
@@ -0,0 +1,137 @@
+import React from 'react';
+import Proptypes from 'prop-types';
+import styled from 'styled-components';
+
+const SocialWrapper = styled.a`
+  display: inline-flex;
+`;
+
+const StyledSvg = styled.svg`
+  display: inline-block;
+`;
+
+const SocialType = ({ type, url, size, description }) => {
+  const SVGOriginalSize = 48;
+  const SVGOriginalSizeWithBorder = 50;
+
+  return (
+    <SocialWrapper href={url}>
+      {type === 'twitter' && (
+        <StyledSvg
+          role="img"
+          aria-labelledby="twitter-title"
+          xmlns="http://www.w3.org/2000/svg"
+          width={size}
+          height={size}
+          viewBox={`0 0 ${SVGOriginalSize} ${SVGOriginalSize}`}
+        >
+          <title id="twitter-title">{description}</title>
+          <g id="Box_Twitter" transform="translate(606 -203)">
+            <rect
+              id="twitterID"
+              data-name="twitter-name"
+              width={SVGOriginalSize}
+              height={SVGOriginalSize}
+              rx="9"
+              transform="translate(-606 203)"
+              fill="#2caae1"
+            />
+            <path
```

Comment:
the svg file could be in a separated file and be imported. Code would be cleaner.